versioned_docs/version-2.0/how_to_guides/tracing/log_tool_call_traces.mdx

```diff
@@ -0,0 +1,46 @@
+---
+sidebar_position: 9
```

Comment:
Not sure which position is best. This is the same number as the retriever tracing page.